### ISSUE TITLE
GoDaddy domains fail

### ISSUE THREAD

**Issue by nmalcolm** [weppos/whois-parser#61]
I believe this was supposed to be fixed in 1.0.1, but it's failing for me. I've reinstalled the gem multiple times with the same result.

```
$ gem list --local | grep whois-parser
whois-parser (1.0.1)
```
Example code:

```
require 'whois-parser'

record = Whois.whois('university.com')
parser = record.parser

puts parser.expires_on
```

The result is nil. It works fine for non-GoDaddy domains as far as I've tested.

**Comment by SebastianJ** [comment]
It seems to happen with other ICANN domains as well.

The issue seems to be that the parser is scanning for "Registrar Registration Expiration Date" whereas the actual response is "Registry Expiry Date" when trying to populate the "expires_on" field.

**Comment by gayanW** [comment]
When this will be fixed?

**Comment by weppos** [comment]
@gayanW feel free to provide a PR

**Comment by sairam** [comment]
This issue can be closed now since I am able to get this working with 1.1.0 version of whois-parser

**Comment by weppos** [comment]
Closed by #60
